Lib/test/test_sqlite3/test_cli.py

```diff
@@ -200,5 +205,74 @@ def test_color(self):
             self.assertIn('\x1b[1;35mOperationalError (SQLITE_ERROR)\x1b[0m: '
                           '\x1b[35mnear "sel": syntax error\x1b[0m', err)
 
+@requires_subprocess()
```

Comment:
```suggestion

@requires_subprocess()
```

Two blank lines to separate classes.